src/main/java/org/broadinstitute/consent/http/service/ElasticSearchService.java

```diff
@@ -173,6 +173,12 @@ public StudyTerm toStudyTerm(Study study) {
     term.setPiName(study.getPiName());
     term.setPublicVisibility(study.getPublicVisibility());
 
+    findStudyProperty(
+        study.getProperties(), "dbGaPPhsID"
```

Comment:
Just curious, where is the source of truth for this property name? That capitalization is pretty wacky!